### PR TITLE
Recognize alternative form of sigmoid in logprob inference

### PR DESCRIPTION
**Pull request by ricardoV94**
----
:books: Documentation preview :books:: https://pymc--6978.org.readthedocs.build/en/6978/

### CODE REVIEW

File: tests/logprob/test_transforms.py [hunk 1]
```diff
@@ -1127,31 +1127,34 @@ def test_cosh_rv_transform():
...
-    "log10": (pt.log10, lambda x: pt.log(x) / pt.log(10)),
-    "exp2": (pt.exp2, lambda x: pt.exp(pt.log(2) * x)),
-    "expm1": (pt.expm1, lambda x: pt.exp(x) - 1),
-    "sigmoid": (pt.sigmoid, lambda x: 1 / (1 + pt.exp(-x))),
-}
-
-
-@pytest.mark.parametrize("transform", TRANSFORMATIONS.keys())
-def test_special_log_exp_transforms(transform):
+TRANSFORMATIONS = {}
```
Comment (LukeLB) [inline tests/logprob/test_transforms.py]: I don't think this dictionary is needed anymore

File: tests/logprob/test_transforms.py [hunk 2]
```diff
@@ -1127,31 +1127,34 @@ def test_cosh_rv_transform():
...
+
+
+@pytest.mark.parametrize(
```
Comment (LukeLB) [inline tests/logprob/test_transforms.py:1130]: The parameritisation is much cleaner I don't know why I didn't do it before

**Review by LukeLB — CHANGES_REQUESTED** [review]
All looks good to me! Just need to remove that empty dictionary

File: tests/logprob/test_transforms.py (hunk 1 above)
Comment (ricardoV94) [reply tests/logprob/test_transforms.py]: Thanks for the catch

**Pull request merged**